### PR TITLE
Update cloud spaces articles [HC-1427, HC-1428, HC-1429, HC-1430, HC-1432]

### PR DESCRIPTION
**Pull request by Renat0Ribeir0**

### CODE REVIEW

File: content/Arduino Cloud/Spaces/Add-members-to-a-space.md [hunk 1]
```diff
@@ -9,27 +9,29 @@ Learn how to invite new members to a shared space in Arduino Cloud.
...
-     ![The shareable link and URL.](img/invite-code.png)
+6. **Invite with a link/code:**
 
-     Students can join using the code by going to [app.arduino.cc](https://app.arduino.cc/) and selecting "Join space" in the space dropdown.
+   * Share the link (click on "link" to copy) with a student.
 
-     ![The space selection dropdown.](img/dropdown-join-space.png)
+   * Or share the code (for minor students only). If the minor student does not belong to any Educational space, an orange banner will be shown in the sidebar when the student opens Arduino Cloud. Clicking on it will open a text box that allows the student to join the space by inserting the code.
+
+     ![The shareable link and URL.](img/cloud-invite-link.png)
```
Comment (seaxwi) [inline content/Arduino Cloud/Spaces/Add-members-to-a-space.md:34]: I was a bit confused by this being two different steps (5 & 6) at first. I think they're different means of acheiving the end result of adding/inviting members. So for me, having this information grouped under step 5 makes sense.

File: content/Arduino Cloud/General Cloud/Sign-up-for-an-Arduino-Cloud-plan.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Learn how to review and sign up for Arduino Cloud plans.
 5. Click **Get started** or **Purchase** to proceed.
 6. If you're not signed into an Arduino account, you will be asked to do so.
 7. If you selected a paid plan, proceed with the checkout instructions.
-8. Start using the [IoT Cloud](https://app.arduino.cc/) and [Web Editor](https://create.arduino.cc/editor) apps right away, or check the [Resources tab](https://cloud.arduino.cc/home/resources) for help getting started.
+8. Start using the [IoT Cloud](https://app.arduino.cc/) and [Web Editor](https://create.arduino.cc/editor) apps right away, or check the [Resources tab](https://app.arduino.cc/resources) for help getting started.
```
Comment (seaxwi) [inline content/Arduino Cloud/General Cloud/Sign-up-for-an-Arduino-Cloud-plan.md:19]:
```suggestion
8. Start using [Arduino Cloud](https://app.arduino.cc/) and [Web Editor](https://create.arduino.cc/editor) apps right away, or check the [Resources tab](https://app.arduino.cc/resources) for help getting started.
```

**Review by seaxwi — APPROVED** [review]
Two comments but otherwise LGTM 👍

File: content/Arduino Cloud/Spaces/Add-members-to-a-space.md (hunk 1 above)
Comment (Renat0Ribeir0) [reply content/Arduino Cloud/Spaces/Add-members-to-a-space.md]: [c17062e](https://github.com/arduino/help-center-content/pull/311/commits/c17062ed72f9198b5ab148e94c8dafa816ae0b4e)

**Pull request merged**